### ISSUE TITLE
Error using .time()

### ISSUE THREAD

**Issue by MateusJSouza** [colinhacks/zod#3405]
I'm trying to validate the date and time, but when I use .time() I get this error. Is there any alternative to using .time() to validate the time?

```js
export const appointmentSchema = z.object({
  clientHashId: z.string({ required_error: 'Client ID is required' }),
  employeeHashId: z.string({
    required_error: 'Employer ID is required',
  }),
  appointmentDate: z.coerce.date(),
  appointmentTime: z.string().time(),
})
```

**Error:** _The 'time' property does not exist in the 'ZodString' type._

To validate the date/time I'm using date-fns in the form submit, but could it be validated in zod as well or not necessarily?

Thank you in advance for your help.

**Comment by colinhacks** [comment]
This functionality is now available in `zod@beta`

**Comment by colinhacks** [comment]
This has landed in Zod 3.23.

https://github.com/colinhacks/zod/releases/tag/v3.23.0
